OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan dark mode for all dashboard modals

# Description
Implemented dark mode for all dashboard modals. 

## Related PRS (if any):
This frontend PR is related to the dev backend

## Main changes explained:
-Dark mode on modals: **header, timelog, leaderboard, badge, and weekly summary**
-Updated loading skeletons to be in dark mode


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard
6. verify that modals are working properly across all components (**header, timelog, leaderboard, badge, and weekly summary**)

## Screenshots or videos of changes:
![Screenshot 2024-05-01 203647](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/f1dd21a0-3d36-495a-a8ef-91a75e628b7e)
![Screenshot 2024-05-01 203717](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/aa40dd47-508c-4254-921f-bd98d99f6a84)


## Note:
Please request changes if you find any modal that is not working in dark mode
